src/store/facade/GroceryStore.java

```diff
@@ -49,9 +52,12 @@
 		 * @param member - the Member being added
 		 * @return the new member's ID
 		 */
-		public String addMember(Member member) {
-			members.add(member);
-			return members.get(members.size() - 1).getId();
+		public String add(Member member) {
```

Comment:
So this might be streamlined but for our benefit could we have this methods name be more specific about it's functionality?

Reply:
We technically could, but I implemented the GroceryStoreList interface (since Brahma was calling for interfaces - if I understood him correctly) and every collection has the same add method. That's why there's no addMember() vs. addProduct() vs. addOrder() method... Take a peek at the GroceryStoreList interface (under store.facade).

Reply:
Hhhhmmmmmmmmm.... Well if it's a choice of my preference and brahmas specifications we should go with the specifications.

Reply:
Don't worry, it will be easy enough to read: membersList.add(), productsList.add(), and ordersList.add(). Pretty legible.